ground/src/test/java/com/google/android/ground/ui/datacollection/DataCollectionFragmentTest.kt

```diff
@@ -67,41 +69,44 @@ import org.robolectric.shadows.ShadowToast
 @RunWith(RobolectricTestRunner::class)
 class DataCollectionFragmentTest : BaseHiltTest() {
 
+  @Inject lateinit var activateSurvey: ActivateSurveyUseCase
+  @Inject lateinit var fakeRemoteDataStore: FakeRemoteDataStore
   @BindValue @Mock lateinit var submissionRepository: SubmissionRepository
   @Captor lateinit var taskDataDeltaCaptor: ArgumentCaptor<List<TaskDataDelta>>
   lateinit var fragment: DataCollectionFragment
 
   @Before
   override fun setUp() {
     super.setUp()
```

Comment:
this is now just calling super(). can be removed

Reply:
Done